drizzlepac/hlautils/align_utils.py

```diff
@@ -489,20 +493,9 @@ def build_dqmask(self, chip=None):
         # source match algorithm from trying to match multiple sources
         # from one image to a single source in the
         # other or vice-versa.
-        # Create temp DQ mask containing all pixels flagged with any value EXCEPT 256
-        non_sat_mask = bitfield_to_boolean_mask(dqarr, ignore_flags=256+2048)
-
-        # Create temp DQ mask containing saturated pixels ONLY
-        sat_mask = bitfield_to_boolean_mask(dqarr, ignore_flags=~(256+2048))
+        # Create temp DQ mask containing all pixels flagged with any value EXCEPT 512 (bad ref pixel)
```

Comment:
Update "grow out" comment as it refers to saturated pixels only.

Reply:
The comment was completely removed since it referred to code that had been removed entirely.  Fixed in 6b78487 .